agent/recorder-agent.c

```diff
@@ -342,7 +342,7 @@ main (int argc, char *argv[])
   g_autoptr (GApplication) app = NULL;
 
   app = G_APPLICATION (g_object_new (HWANGSAE_TYPE_RECORDER_AGENT,
-          "application-id", "org.hwangsaeul.Hwangsae1",
+          "application-id", "org.hwangsaeul.Hwangsae2",
```

Comment:
In order to allow both `hwangsae-relay-agent` and `hwangsae-recorder-agent` to work in the same machine I've changed the D-Bus name, but probably there is some other preferred approach.

Reply:
Actually, the number is about ABI version. For this case, `application-id` should be `org.hwangsaeul.Hwangsae1.RecorderAgent` or similar.